### DISCUSSION TITLE
0.0.3

### DISCUSSION THREAD

**Discussion by psemiletov** [psemiletov/drumlabooh#1] [Announcements]
* major documentation update
* fixes, fixes
+ Windows VSTi build
+ JUCE 7.0.7 sync
- Un-needed optimizations those caused crash on build

<hr /><em>This discussion was created from the release <a href='https://github.com/psemiletov/drumlabooh/releases/tag/0.0.3'>0.0.3</a>.</em>

**Comment by pablito331** [comment]
Hello friend, I tried your plugin and it is really very good. Could you pass me more information about it, for example, about the midimap, and about more details for creating kits?

**Reply by psemiletov** [reply]
Hello! The drumkit format is very simple and described at [https://github.com/psemiletov/drum_sklad](https://github.com/psemiletov/drum_sklad). The midimap currently hardcoded and based on the order from the drumkit description file - first sample is the base note (MIDI 36 by default,  can be changed at Drumlabooh GUI).
